tests/python/unittest/test_runtime_trace.py

```diff
@@ -87,11 +95,14 @@ def check_expr_sum(dtype):
 
 
 def test_trace_expr_sum_args():
+    """Test trace expression sum arguments"""
+
     @tvm.register_func("tvm.tir.trace_silent")
-    def silent(*args):
+    def silent(*_args):  # pylint: disable=unused-variable
         return
 
     def check_expr_sum(dtype):
+        # pylint: disable=invalid-name
```

Comment:
consider removing in favor of updating good-names

Reply:
the unused variable is silent()